### PR TITLE
Functionality of Application

### PR DESCRIPTION
**Pull request by robinbenitezmora**
As a developer team we just gave the final steps into complete functionality of our application so that the user can have an good experience.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+</head>
+<body>
+  <div class="container">
+    <header>
+      <div class="logo"></div>
+      <nav>
+        <a href="#shows" class="shows">Shows</a>
+        <a href="#episodes" class="episodes">Episodes</a>
+        <a href="#people" class="people">People</a>
+      </nav>
```
Comment (DeliceLydia) [inline src/index.html:18]: - [ ] I suggest that you remove `links` that do not have take us anywhere

File: README.md [hunk 2]
```diff
@@ -1 +1,83 @@
...
+
+## Acknowledgments
+
+-I thank Microverse for its documentation.
+
+-Thanks to my coding partner Armando Comellas for helping with your excellent support in this project.
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed
```
Comment (DeliceLydia) [inline README.md:83]: - [ ] Please add in the README video presentation that consists:
-  Presentation of the features of the project.
-  The video should contain a clear and objective description of the project in professional English. The topic and purpose of the project are should be made clear.
-  All the members should appear in the video and talk about the project.
-  The video should contain a highlight about a piece of the code you find interesting or are proud of making.

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hello @robinbenitezmora and @armandocomellas1, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there! Highlights for your project:
- Application is working well
- Awesome README


## Suggested changes
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md (hunk 2 above)
Comment (armandocomellas1) [reply README.md]: Got it thank you so much @DeliceLydia  it is done and added!

**Review by DeliceLydia — APPROVED** [review]
## Hi Team, 

In my opinion, your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


### Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**